### DISCUSSION TITLE
Admin UI relations

### DISCUSSION THREAD

**Discussion by thal-sh** [pocketbase/pocketbase#4325] [Q&A] [answered]
Let's say i have a cart table and products table, normally the cart would have a relationship with products, isnt there any way for the images to be preferred and shown so that my client would be more comfortable instead of showing these random ids

**Comment by ganigeorgiev** [comment, marked as answer]
You could try to mark the products file field as "Presentable" from the file field options.

![presentable_screenshot](https://i.imgur.com/cnU7xNm.png)

**Comment by thal-sh** [comment]
Hmm interesting, it worked, btw is there a way for example when the relation id is shown, i could click on it and itll take me to the record u get me?

**Reply by ganigeorgiev** [reply]
> btw is there a way for example when the relation id is shown, i could click on it and itll take me to the record u get me?

Not directly (_there is an "Edit" btn once you click on the detailed view_) but I'll consider adding a shortcut with the next major release.

**Comment by thal-sh** [comment]
I don't see any edit button, and before I close this discussion I wanna ask, let's say I have products that will be stored in a cart then the cart will be in an order table (all referenced). If I do the presentable it'll only show from cart but on order it'll show the ID of the image (I want the image to show).

**Reply by thal-sh** [reply]
![Screenshot_20240210-190656](https://github.com/pocketbase/pocketbase/assets/99901847/ec35c915-e3d7-45c6-98f0-144ffb39e017)

**Reply by ganigeorgiev** [reply]
The presentable for relations works only when it is a direct relation (there is an existing feature request https://github.com/pocketbase/pocketbase/issues/3666 to pass it recursively but I have some performance concerns and it is left for later consideration). 

The "Edit" button is available in the relation picker popup.

**Comment by thal-sh** [comment]
alright thanks ganigeorgiev, it would be cool to add the id record lookup on next release :)
